rbac/ledger_sync/inbound/rbac_transactions.py

```diff
@@ -121,6 +125,7 @@ def add_metadata(inbound_entry, message):
         dictionary from data and removes fields that are in the message.
     """
     message_keys = [f[0].name for f in message.ListFields()]
+    message_keys.append("metadata")
```

Comment:
metadata is already in the list of message_keys, this line isn't necessary.

Reply:
You finally got me here buddy. My bad, I missed checking all the available message_keys before appending.  👍